modules/mongodb/mongodb_test.go

```diff
@@ -0,0 +1,87 @@
+package mongo
+
+import (
+	"testing"
+	"time"
+
+	"github.com/netdata/go.d.plugin/agent/module"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNew(t *testing.T) {
+	job := New()
+	assert.Implements(t, (*module.Module)(nil), job)
+	assert.Equal(t, time.Duration(defaultTimeout), job.Timeout, "wrong timeout")
+	assert.Equal(t, defaultUri, job.Uri, "wrong timeout")
```

Comment:
No need to test that if we remove both default* constants.

Reply:
done